model_security_adjust_cft/security/sale_security.xml

```diff
@@ -16,5 +16,12 @@
         <field eval="[(4,ref('group_sale_salesman_team_leads'))]" name="groups"/>
     </record>
 
+    <record id="sale_order_line_see_team" model="ir.rule">
+        <field name="name">Team Orders Line</field>
+        <field name="model_id" ref="sale.model_sale_order_line" />
+        <field name="domain_force">[(1,'=',1)]</field>
```

Comment:
`[('order_id.section_id.member_ids','in',[user.id])]` to be consistent with "Team Orders" record rule.